doc/source/whatsnew/v0.18.0.txt

```diff
@@ -312,6 +312,24 @@ New Behavior:
    s.index
    print(s.to_csv(path=None))
 
+Latex Represenation
+^^^^^^^^^^^^^^^^^^^
+
+``DataFrame`` has gained a ``_repr_latex_`` method in order to allow
+for conversion to latex in a ipython/jupyter notebook using nbconvert. (:issue:`11778`)
+
+Note that this must be activated by setting the option ``display.latex.repr`` to ``True`` (issue:`12182`)
+
+.. ipython:: python
+
+   df = pd.DataFrame({'a': [1, 2, 3]})
+   df._repr_latex_()
+   with pd.option_context('display.latex.repr', True):
+       print(df._repr_latex_())
```

Comment:
Yeah, I see what you're saying, it is contrived.  I was just trying to think of some example - for some reason I always forget the options syntax.  Maybe instead

```
For example, if you have a jupyter notebook you plan to convert to latex using nbconvert, place the statement ``pd.set_option('display.latex.repr', True)`` in the first cell to have the contained DataFrame output also stored as latex.
```